app/main.py

```diff
@@ -1,6 +1,41 @@
 from typing import Callable
+import functools
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    cache_storage = {}  # Dictionary to store results of completed runs
+
+    @functools.wraps(func)
+    def wrapper(*args) -> Callable:
+        if args in cache_storage:
+            print("Getting from cache")
```

Comment:
The print statement is generally used for debugging purposes. Its usage in production code is discouraged, as it may reveal internal workings of the code to the user. Consider replacing it with logging or removing it altogether.